contracts/adapters/SFCToGovernable.sol

```diff
@@ -2,48 +2,73 @@ pragma solidity ^0.5.0;
 
 import "../model/Governable.sol";
 
+/// @dev SFC is an interface which implements some functions from the SFC required for the Governance contract.
```

Comment:
SFC is representation of the network SFC contract for the purpose of the Governance contract. It provides weights of individual voters.